packages/react/menu/src/Menu.tsx

```diff
@@ -236,10 +346,8 @@ const MenuContentImpl = React.forwardRef((props, forwardedRef) => {
     <PortalWrapper>
```

Comment:
I get the feeling that once the modality stuff is in, sub-menus will become non-modal, so this wouldn't be true anymore. And it kinda makes sense, if you're able to focus or hover the parent menu, then screen readers should also be able to "see" them. So I would say for now, no need to do anything specific for it in this branch.

Reply:
> if you're able to focus or hover the parent menu, then screen readers should also be able to "see" them

I'm saying that currently they can't "see" them. If a sub menu opens, the parent will become hidden even though it's still open because this effect will run in the submenu causing everything outside of it to be hidden.

I agree non-modal stuff might solve some of this but that's not ready yet and we need to release with an accessible thing so should we need to prevent this block from running in sub menus?

Reply:
Yes exactly, that's why I said sub-menus would be non-modal.
Non-modal means that it everything is still visible to SR users.
So yeah in the meantime you just have to guard the effect in sub-menus for now 👍